### PR TITLE
W-16804223-document-configuration-panel-fa

### PR DESCRIPTION
**Pull request by failup**

### CODE REVIEW

**Review by dpopovms — APPROVED** [review]

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 1]
```diff
@@ -89,7 +89,35 @@ To find information about more connectors, see xref:connectors::introduction/int
 // end::connectors-ref[]
 //
 
+// tag::connection-configuration-panel[]
 
+The Connection Management Configuration Panel provides an interface that allows you to easily configure connections to third-party systems directly from the UI.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
Use the Connection Management Configuration Panel to easily configure connections to third-party systems directly from the UI. 
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 2]
```diff
@@ -89,7 +89,35 @@ To find information about more connectors, see xref:connectors::introduction/int
 // end::connectors-ref[]
 //
 
+// tag::connection-configuration-panel[]
 
+The Connection Management Configuration Panel provides an interface that allows you to easily configure connections to third-party systems directly from the UI. 
+
+In this  panel, you can create, edit, delete, and test connections directly within the ACB interface, automatically populate connection fields using metadata provided by the connector, and save and manage connection configurations without switching to the XML view. To open the configuration panel:
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
In this panel, you can create, edit, delete, and test connections directly within the ACB interface, automatically populate connection fields using metadata provided by the connector, and save and manage connection configurations without switching to the XML view. 

To open the configuration panel: 
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 3]
```diff
@@ -89,7 +89,35 @@ To find information about more connectors, see xref:connectors::introduction/int
...
 //
 
+// tag::connection-configuration-panel[]
 
+The Connection Management Configuration Panel provides an interface that allows you to easily configure connections to third-party systems directly from the UI. 
+
+In this  panel, you can create, edit, delete, and test connections directly within the ACB interface, automatically populate connection fields using metadata provided by the connector, and save and manage connection configurations without switching to the XML view. To open the configuration panel: 
+
+* Click on an existing component, or add a new one.
+* Click the *+* icon:
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
* Click the *+* icon.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 4]
```diff
@@ -89,7 +89,35 @@ To find information about more connectors, see xref:connectors::introduction/int
...
+* Click on an existing component, or add a new one.
+* Click the *+* icon: 
++
+This action displays the configuration panel, for example:
+
+image:anypoint-code-builder::acb-configuration-panel-general.png["Configuration panel for connectors"] 
+
+From here, you can:
+
+* Configure a connection for this project.  You can select a previously saved connection on the Connection field, where the form is pre-populated with existing configuration data.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
* Configure a connection for this project.
+
You can select a previously saved connection in the *Connection* field and the form is pre-populated with existing configuration data.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 5]
```diff
@@ -89,7 +89,35 @@ To find information about more connectors, see xref:connectors::introduction/int
...
+
+NOTE: Changes impact all components using this connection.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc:114]:
```suggestion
+
NOTE: Changes impact all components using this connection.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 6]
```diff
@@ -89,7 +89,35 @@ To find information about more connectors, see xref:connectors::introduction/int
...
+
+* Update connection details without manually editing the XML code.
+
+* Test the connection directly within the panel to verify that it is correctly configured. The test functionality helps prevent errors by allowing users to confirm that the connection is valid before committing changes.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
* Test the connection directly within the panel to verify it's correctly configured.
+
The test functionality helps prevent errors by allowing users to confirm that the connection is valid before committing changes.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 7]
```diff
@@ -89,7 +89,35 @@ To find information about more connectors, see xref:connectors::introduction/int
...
+
+* Delete an existing connection when it's no longer needed. A confirmation message is displayed, listing any affected components that use the connection.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
* Delete an existing connection if it's no longer needed.
+
A confirmation message shows any affected components that use the connection.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 8]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
 
 //
 // tag::check-connection[]
-Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically have connection configurations that you can test from the configuration XML. For example, to test an HTTP Listener connection, click *Test Connection* in the configuration XML:
+
+Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically have connection configurations that you can test
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically include connection configurations that you can test.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 9]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
 
 //
 // tag::check-connection[]
-Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically have connection configurations that you can test from the configuration XML. For example, to test an HTTP Listener connection, click *Test Connection* in the configuration XML:
+
+Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically have connection configurations that you can test 
+
+== Test Connection from The Configuration Panel
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
== Test Connection from the Configuration Panel
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 10]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
...
+
+You can text the connection from the connectors *Configuration panel*. For example, to test an HTTP Listener connection:
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
You can test the connection from the connectors *Configuration* panel. For example, to test an HTTP Listener connection: 
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 11]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
...
 // tag::check-connection[]
-Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically have connection configurations that you can test from the configuration XML. For example, to test an HTTP Listener connection, click *Test Connection* in the configuration XML:
+
+Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically have connection configurations that you can test 
+
+== Test Connection from The Configuration Panel
+
+You can text the connection from the connectors *Configuration panel*. For example, to test an HTTP Listener connection: 
+
+image:anypoint-code-builder::acb-configuration-panel-test.png["Configuration panel for connectors"]
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc:321]: In the alt text, mention that it's showing the HTTP Listener connection?

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 12]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
...
+Most connectors provide an operation, such as the HTTP Listener operation, that triggers the flow to start. These operations typically have connection configurations that you can test 
+
+== Test Connection from The Configuration Panel
+
+You can text the connection from the connectors *Configuration panel*. For example, to test an HTTP Listener connection: 
+
+image:anypoint-code-builder::acb-configuration-panel-test.png["Configuration panel for connectors"] 
+
+[calloutlist]
+.. Click the *+* icon in the connector to open the configuration panel
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
.. Click the *+* icon on the connector to open the configuration panel.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 13]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
...
+.. After filling in the required fields, click *Test connection*.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
.. After filling in the required fields, click *Test Connection*.
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 14]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
...
+
+The status bar shows the progress:
+
+* *Verifying connection* indicates that the test is in progress.
+* *Connection is valid* indicates a successful connection. 
+* *Invalid Connection Got status code: 500 when trying to resolve a Mule Runtime operation* indicates a connection error.
+
+
+== Test Connection from The configuration XML
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc]:
```suggestion
== Test Connection from the Configuration XML
```

File: modules/ROOT/pages/_partials/acb-component-info.adoc [hunk 15]
```diff
@@ -274,7 +302,29 @@ Your configuration XML file now looks similar to the following:
...
+
+You can test the connection from the configuration XML. For example, to test an HTTP Listener connection, click *Test Connection* in the configuration XML:
 
 image::http-listener-test-connection.png["HTTP Listener Test Connection link in the configuration XML"]
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-component-info.adoc:338]: Nice alt text!

**Pull request merged**